app/kuma-cp/cmd/root.go

```diff
@@ -61,6 +67,7 @@ func newRootCmd() *cobra.Command {
 	// root flags
 	cmd.PersistentFlags().StringVar(&args.logLevel, "log-level", kuma_log.InfoLevel.String(), kuma_cmd.UsageOptions("log level", kuma_log.OffLevel, kuma_log.InfoLevel, kuma_log.DebugLevel))
 	cmd.PersistentFlags().StringVar(&args.outputPath, "log-output-path", args.outputPath, "path to the file that will be filled with logs. Example: if we set it to /tmp/kuma.log then after the file is rotated we will have /tmp/kuma-2021-06-07T09-15-18.265.log")
+	cmd.PersistentFlags().StringVar(&args.logFormat, "log-format", args.logFormat, "specify logformat, json | logfmt (default)")
```

Comment:
You should set `kuma_log.LogFmt.String()` as a default param and add the kuma_cmd.UsageOptions which should add the right help info (See the logLevel line)